src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -122,6 +122,21 @@ export default {
 			 * Comparing two crumbs somehow does not work, so we use the indices.
 			 */
 			hiddenIndices: [],
+
+			/**
+			 * This is the props of the middle Action menu
+			 * that show the ellipsised breadcrumbs
+			 */
+			menuBreadcrumbProps: {
```

Comment:
@ChristophWurst, @korelstar, @raimund-schluessler  clean enough ?
We use the prop as object here, so it's passed as a reference through the Breadcrumb and therefore we can toggle it directly from `this.menuBreadcrumbProps.open = true/false` :)

Reply:
Fine with me.

Reply:
A little review then? :pray:

Reply:
> A little review then? 🙏

Sorry, I don't have time at the moment, but I will try to have a more detailed look in the evening.

Reply:
I'm having a bit of a hard time grasping this, but I think it's fine

Reply:
yeah, that part I got :) I was just confused to suddenly see breadcrumbs here when it originally was about actions